here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PostgresStorage.java

```diff
@@ -450,6 +439,50 @@ synchronized void initStorage(@NotNull PsqlStorage.Params params, @NotNull IoHel
     }
   }
 
+  private void initStoragePlv8(
+      PsqlStorage.@NotNull Params params, @NotNull IoHelp ioHelp, Statement stmt, PsqlConnection conn)
+      throws SQLException {
+    final Plv8Env plv8Env = new Plv8Env();
```

Comment:
Ensure proper resource management by using try-with-resources for the Statement and PsqlConnection objects to ensure they are closed even in the case of an exception.(For both initStoragePlv8 & initStoragePlpgsql)

Something like this
```
try (Statement stmt = conn.createStatement()) {
    // existing code
} catch (SQLException e) {
    // handle the exception
}
```

Reply:
This method does not create the statement or any other closable resources. It is already handled by the caller.